taxonomy/models.py

```diff
@@ -0,0 +1,31 @@
+"""Models for lore taxonomy application"""
+
+from django.db import models
+
+
+class Vocabulary(models.Model):
+    repository = models.ForeignKey("Repository")
+    name = models.TextField()
+    description = models.TextField()
+    required = models.BooleanField()
+    type = models.CharField(max_length=1, choices=(
```

Comment:
I agree with Carson about the name. Also, I made a mistake; it's not a keyword, but it _is_ a [built-in function](https://docs.python.org/2/library/functions.html) in Python.